packages/terra-navigation-side-menu/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-navigation-side-menu",
+  "main": "lib/NavigationSideMenu.js",
+  "version": "0.0.0",
+  "description": "A structual component for nesting navigational items within the content section of the terra-menu-layout.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-navigation-side-menu",
+    "NavigationSideMenu",
+    "Navigation",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.11.0"
+  },
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^2.10.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-clinical-action-header":"^2.0.0",
+    "terra-content-container":"^2.0.1",
+    "terra-icon":"^2.0.1",
+    "terra-list":"^2.0.1"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rimraf lib",
+    "compile:build": "babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "props-table": "props-table ./src/NavigationSideMenu.jsx --out-dir ./docs/props-table",
+    "test": "npm run test:jest && npm run test:wdio",
+    "test:jest": "jest ./tests/jest/* --config ../../jestconfig.json",
```

Comment:
Wondering if we should just get rid of jest at this point.  Thoughts?

Reply:
Why would we want to drop jest?

Reply:
My brain wasn't working.  Nothing to see here :-/